assets/js/src/notices/notice.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { FC, ReactNode } from 'react';
 import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
 
-const Notice = ({
+type Props = {
+  type: 'success' | 'info' | 'warning' | 'error';
+  children: ReactNode;
```

Comment:
This was string/element/array-of-elements originally. Is ReactNode sufficient? Thinking about it, maybe it is - ReactNode can contain any children, right? Just double checking.

Reply:
Yes!